appveyor.yml

```diff
@@ -43,6 +43,20 @@ before_build:
     - IF "%platform%" == "x64" set generator="Visual Studio 14 2015 Win64"
     - git submodule update --init --recursive
     - git fetch --tags
+    - vcpkg integrate install
+    - vcpkg install fmt:%platform%-windows
+    - git clone https://github.com/equinor/layered-file-protocols.git
+    - ps: mkdir layered-file-protocols/build
+    - ps: pushd layered-file-protocols/build
+    - cmake -G %generator%
+            -DCMAKE_CXX_FLAGS="/D_CRT_SECURE_NO_WARNINGS /EHsc"
+            -DLFP_FMT_HEADER_ONLY=ON
+            -DCMAKE_TOOLCHAIN_FILE=C:/Tools/vcpkg/scripts/buildsystems/vcpkg.cmake ..
```

Comment:
As Release apparently is to be always specified, maybe you need to it here also.